.github/workflows/examples.yaml

```diff
@@ -152,7 +152,7 @@ jobs:
           GALILEO_MODAL_TOKEN_SECRET: ${{ secrets.GALILEO_MODAL_TOKEN_SECRET }}
         run: |
           modal token set --token-id ${{ env.GALILEO_MODAL_TOKEN_ID }} --token-secret ${{ env.GALILEO_MODAL_TOKEN_SECRET }}
-          modal run ./scripts/populator_cuda.py::run --notebook-path examples/image_classification/Image_Classification_using_PyTorch_and_🔭_Galileo.ipynb
+          modal run ./scripts/populator_cuda.py::run --notebook-path examples/image_classification/Image_Classification_using_PyTorch_and_🔭_Galileo_IF_data.ipynb
```

Comment:
this should not be removed because its for example runs.

Reply:
@anthonycorletti I didn't add an inference notebook by itself, I just added inference cells at the end of the normal notebooks. Also, renamed our notebooks since the names were a bit weird, it's now
- Galileo_HF_data for HuggingFace
- Galileo_IF_data for ImageFolder
- Galileo_DF_data for DataFrame

Reply:
I see, these will now fail in cloud since they have inference. Okay will fix that